### PR TITLE
Remove hardcoded AAT value for draft encryption key

### PR DESCRIPTION
**Pull request by tchow8**
Remove hardcoded encryption key value. This is passed in via vault.

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/functionaltest/MockSupport.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
     static final String USER_ID = "1";
     static final String CASE_WORKER_USER_ID = "2";
     static final String USER_EMAIL = "test@test.com";
-    static final String ENCRYPTED_USER_ID = "OVZRS2hJRDg2MUFkeFdXdjF6bElfMQ==";
+    static final String ENCRYPTED_USER_ID = "ZW5jcnlwdGlvbktleV8x";
```
Comment (tchow8) [inline src/test/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/functionaltest/MockSupport.java]: This is the base64 encoding of `encryptionKey_1` the new default value (+ unique identifier) for the draft store encryption key

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -65,7 +65,7 @@ draft:
       encryption:
         key:
           template: "%s_%s"
-          value: ${DRAFT_STORE_API_ENCRYPTION_KEY_VALUE:9VQKhID861AdxWWv1zlI}
+          value: ${DRAFT_STORE_API_ENCRYPTION_KEY_VALUE:encryptionKey}
```
Comment (yarpo) [inline src/main/resources/application.yml]: why not `replace_me`?

Comment (tchow8) [reply src/main/resources/application.yml]: Can be, I thought with consistency the -aat.yml had replace_me placeholders whilst this main .yml just had some text ones. But can make a change if you prefer?

**Comment by hosainnet** [comment]
Merge up?

**Comment by tchow8** [comment]
Closing due to old age. Not a priority.

**Pull request closed without merging**